src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -460,13 +487,19 @@ template <typename InterfaceType> void RadioSpinel<InterfaceType>::HandleReceive
     otError        error = OT_ERROR_NONE;
     uint8_t        header;
     spinel_ssize_t unpacked;
+    spinel_iid_t   iid;
 
     LogSpinelFrame(mRxFrameBuffer.GetFrame(), mRxFrameBuffer.GetLength(), false);
     unpacked = spinel_datatype_unpack(mRxFrameBuffer.GetFrame(), mRxFrameBuffer.GetLength(), "C", &header);
 
-    VerifyOrExit(unpacked > 0 && (header & SPINEL_HEADER_FLAG) == SPINEL_HEADER_FLAG &&
-                     SPINEL_HEADER_GET_IID(header) == 0,
-                 error = OT_ERROR_PARSE);
+    iid = SPINEL_HEADER_GET_IID(header);
+    if (!IsFrameForUs(iid))
+    {
+        otLogDebgPlat("Discarding spinel message with IID=%u", iid);
```

Comment:
I'm not sure if this log is needed. I think it might print too often making logs hard to read. For 2 interfaces it is going to print for almost 50% of the frames, when using 3 interfaces almost 130% of frame count.

Reply:
It can be useful in some cases while debugging but agree it'll print more often with multiple hosts hence removed it.